app/src/test/java/io/swagslash/settlersofcatan/RobberTest.java

```diff
@@ -19,43 +24,55 @@
     /*
     Unit Tests of the Robber Class
      */
+
+    Board b;
+
+    @Before
+    public void init() {
+        ArrayList<String> listOfNames = new ArrayList<String>();
+        listOfNames.add("Player  1");
+        listOfNames.add("Player  2");
+        listOfNames.add("Player  3");
+        b = new Board(listOfNames, true, 10);
+        b.setupBoard();
+    }
+
     @Test
-    public void TestChoosePlayer(){
-        Robber rob = new Robber(new Hex(null, Hex.TerrainType.FIELD, new AxialHexLocation(1,1)));
-
-       try {
-           Player p = rob.choosePlayer(new Player(null, 0, Color.BLUE, "Player 1"),
-                   new Player(null, 1, Color.RED, "Player 2"),
-                   new Player(null, 2, Color.WHITE, "Player 3"),
-                   new Player(null, 3, Color.GREEN, "Player 4"));
-           Assert.assertTrue(true);
-       }
-       catch (Exception e){
+    public void TestChoosePlayer() {
+        Robber rob = new Robber(new Hex(null, Hex.TerrainType.FIELD, new AxialHexLocation(1, 1)));
+
+        try {
+            Player p = rob.choosePlayer(new Player(null, 0, Color.BLUE, "Player 1"),
+                    new Player(null, 1, Color.RED, "Player 2"),
+                    new Player(null, 2, Color.WHITE, "Player 3"),
+                    new Player(null, 3, Color.GREEN, "Player 4"));
+            Assert.assertTrue(true);
```

Comment:
Des wird immer durchlaufen, weg mit dem assertTrue(true)
und kein try/catch in Tests, wenn a exception kommt dann soll der test failen!